pyproject.toml

```diff
@@ -106,6 +106,12 @@ cmd = "ctest -j3 --test-dir build-python"
 description = "Test ITK Python"
 depends-on = ["build-python"]
 
+[tool.pixi.feature.python.tasks.python-exe]
+cmd = '''cp ./build-python/Wrapping/Generators/Python/WrapITK.pth $(python -c "import site; print(next(p for p in site.getsitepackages() if \'site-packages\' in p))") &&
```

Comment:
Will this path stay the same across systems (OS, local vs remote, etc.)?

Reply:
It should. I tested on Linux.